requirements/edx/base.in

```diff
@@ -69,7 +69,7 @@ django-waffle
 django-webpack-loader               # Used to wire webpack bundles into the django asset pipeline
 djangorestframework
 done-xblock
-edx-ace
+edx-ace>=1.4.0                      # Uses the get_action_links method and templatetag for the goal reminder email feature
```

Comment:
This change should not be necessary. It will automatically pull latest when there is no version specified.

Reply:
thanks, removed